utils/elastic_search/elastic_search_utils.py

```diff
@@ -38,12 +39,17 @@ def build_search_query(self, search_text: str, sort_by: str, order: str, search_
             order = 'asc'
 
         primary_sort_key = {}
-        primary_sort_key[sort_by+'.'+sort_fields_data_type[sort_by]] = order
+
+        sort_by_complement = '.keyword' if sort_fields_data_type[sort_by] == 'keyword' else ''
+        primary_sort_key[sort_by+sort_by_complement] = order
+
         sort_order.remove(sort_by)
 
         tie_break_sort_order = sort_order.pop(0)
         tie_break_sort_key = {}
-        tie_break_sort_key[tie_break_sort_order+'.'+sort_fields_data_type[tie_break_sort_order]] = 'asc'
+
+        sort_by_complement = '.keyword' if sort_fields_data_type[tie_break_sort_order] == 'keyword' else ''
```

Comment:
This is already used twice. Maybe we could create a util method in this same file that gets the `sort_by_complement` from a `sort_by` parameter